services/collaboration/README.md

```diff
@@ -114,68 +115,82 @@ curl -X GET http://localhost:8080/api/collaboration/room/user/rooms \
 {
   "status": "Success",
   "data": [
-    "6721a64b0c4d990bc0feee4c"
+    "6724e9d892fb3e9f04c2e280"
   ]
 }
 ```
 
 ---
 
-## Get Room by Room ID
+## Get Rooms by User ID, Room Status, and User's isForfeit status
 
-This endpoint retrieves the details of a room by its room ID.
+This endpoint retrieves the details of rooms associated with the authenticated user, filtered by the specified room
+status.
 
 - **HTTP Method**: `GET`
-- **Endpoint**: `/api/collaboration/room/{roomId}`
+- **Endpoint**: `/user/rooms/roomStatus/{room_status}/isForfeit/{isForfeit}`
```

Comment:
This needs to be prefixed with `/api/collaboration/`.

Could I propose that the route be shortened to `/api/collaboration/rooms` with query params `roomStatus` and `isForfeit` (`/api/collaboration/rooms?roomStatus={roomStatus}&isForfeit={isForfeit}`). Usually path parameters are used only for IDs, this seems kinda weird and very very long 😆